code/game/machinery/doors/firedoor.dm

```diff
@@ -637,6 +637,14 @@ var/global/list/alert_overlays_global = list()
 		B.change_dir(user.dir)
 		qdel(src)
 
+/obj/item/firedoor_frame/attackby(var/obj/item/weapon/C, var/mob/user)
+	if(C.is_wrench(user))
+		user.visible_message("<span class='attack'>\The [user] deconstructs \the [src] with \a [C].</span>",\
```

Comment:
using the attack span is weird here since this isn't really dangerous, notice would be fine, or even no message (deconstructing a chair has no message for instance)